src/components/UserListScreen.js

```diff
@@ -32,7 +32,6 @@ const UserListScreen = ({
       renderItem={({item: user}) => (
         <UserItem user={user} navigation={navigation} />
       )}
-      keyExtractor={user => user.email}
```

Comment:
Flatlist should hve keyExtractor to improve performance

Reply:
Sorry, I made a mistake here. I thought id would be by default use as key. I have fixed it.